### DISCUSSION TITLE
Add ability to reveal Visitor Password when adding password protection

### DISCUSSION THREAD

**Discussion by ericmatthys** [vercel/community#419] [Idea]
No idea too small? :) I always find myself extra paranoid adding this password because there's no way to reveal the text, so a typo could easily be saved. It'd save some stress to add a button to reveal the password, especially since there's no confirm password field. This is relatively common on mobile.

**Comment by wiiiimm** [comment]
@ericmatthys thanks for your feedback. I have added it to our internal feature request list.
